vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/FieldSteps.java

```diff
@@ -88,6 +93,30 @@ public WebElement clearFieldLocatedByUsingKeyboard(SearchAttributes locator)
         return element;
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and check if it
+     * sorted by text with using <b>sortingOrder</b>.
+     * List should have more than 1 elements.
+     * @param sortingOrder The set of sort's rule<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ASCENDING</b> - will check that list sorting alphabetically,
+     * <li><b>DESCENDING</b> - will check that list sorting reverse alphabetically
+     * </ul>
+     * @param locator list of elements
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void doesElementSorted(SearchAttributes locator, OrderSortingType sortingOrder)
+    {
+        List<WebElement> elements = baseValidations.assertIfAtLeastNumberOfElementsExist(
+                String.format("A fields with attributes%1$s", locator), locator, 2);
+        List<String> listText = elements.stream().map(WebElement::getText).collect(Collectors.toList());
+        List<String> sortedList = listText.stream().sorted(OrderSortingType.ASCENDING.equals(sortingOrder)
+                ? Comparator.naturalOrder() : Comparator.reverseOrder()).collect(Collectors.toList());
+        baseValidations.assertIfListsEquals(String.format("List with element sorted%1$s", sortingOrder),
```

Comment:
What is the output of this assertion, does it give a clue on what is wrong?

Reply:
This is how the output looks like now:  ![image](https://user-images.githubusercontent.com/39829222/90502058-f78e7680-e155-11ea-9b60-881f0f744f1d.png)